### ISSUE TITLE
JAWS does not speak the JavaScript alert dialog in Chrome

### ISSUE THREAD

**Issue by mraccess77** [FreedomScientific/VFO-standards-support#16]
### Summary

JAWS does not speak any contents in the JavaScript alert dialog in Chrome.  Nothing is announced when it appears and insert+b does not read the dialog.
 
Example:
1. Go to any page with a JavaScript alert or confirm dialog
2. Press tab
3. Press insert+b
4. Note nothing is announced

### Expected result

> What did you expect to happen?

### Actual result

JAWS should automatically speak the contents of the dialog and standard dialog commands should work.

#### JAWS version and build number
JAWS >=2018

 
#### Operating System and version
Windows 7

#### Browser and version:
Chrome - latest build.

**Comment by stevefaulkner** [comment]
@mraccess77 Thanks for the issue report, in future would appreciate a working test case. have created one https://s.codepen.io/stevef/debug/qVEqLN

**Comment by stevefaulkner** [comment]
@mraccess77 I can reproduce the issue with JAWS 18.0.453 and Chrome Version 62.0.3202.75 (Official Build) (64-bit) on Win 10. I can also reproduce with NVDA 2107.3, which suggests its a chrome issue. I cannot view an accessibility tree for the `alert dialog` using object inspection tools

**Comment by stevefaulkner** [comment]
@mraccess77 filed bug on chrome https://bugs.chromium.org/p/chromium/issues/detail?id=779501, so closing here for now.
